packages/contractkit/src/identity/claims/keybase.ts

```diff
@@ -0,0 +1,65 @@
+import { Address } from '@celo/utils/lib/address'
+import { isLeft } from 'fp-ts/lib/Either'
+import * as t from 'io-ts'
+import { serializeClaim, SignedClaimType, verifySignature } from './claim'
+import { ClaimTypes, now, TimestampType } from './types'
+
+export const KeybaseClaimType = t.type({
+  type: t.literal(ClaimTypes.KEYBASE),
+  timestamp: TimestampType,
+  // TODO: Validate compliant username before just interpolating
+  username: t.string,
+})
+export type KeybaseClaim = t.TypeOf<typeof KeybaseClaimType>
+
+export const keybaseFilePathToProof = `.well-known/celo/`
+export const proofFileName = (address: Address) => `verify-${address}.json`
+export const targetURL = (username: string, address: Address) =>
+  `https://${username}.keybase.pub/${keybaseFilePathToProof}${proofFileName(address)}`
+
+export async function verifyKeybaseClaim(claim: KeybaseClaim, signer: Address) {
```

Comment:
output type seems weird: `error | void` ? maybe `error | true`?

Reply:
also, add a comment specifying the output format interpretation

Reply:
You know how much I struggle with how to express unsuccessful computation. I wanted to avoid throwing and immediately catching errors, so I opted for the soft version of that which is that a successful completion yields no/undefined return. Made the return type explicit (string | undefined)